subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/ArchiveTreePerformanceTest.groovy

```diff
@@ -40,8 +40,8 @@ class ArchiveTreePerformanceTest extends AbstractCrossVersionPerformanceTest {
         result.assertCurrentVersionHasNotRegressed()
     }
 
-    @RunFor(
-        @Scenario(type = PER_DAY, operatingSystems = [LINUX], testProjects = ["archivePerformanceProject"])
+    /*@RunFor(
+            @Scenario(type = PER_DAY, operatingSystems = [LINUX], testProjects = ["archivePerformanceProject"])
```

Comment:
:x: please do not comment the whole test but use `@RunFor([])` instead

Reply:
I fail to see the difference since I'm going to them back anyways, but ok, will do so, since looks like I need to disable more tests anyways.